src/pages/NewChairConsole.js

```diff
@@ -94,32 +97,39 @@ const Records = (props) => {
             const targetTypes = (v.type === 'Chairperson' || v.type === 'DAC');
             return belongsToUser && targetTypes;
           })(electionInfo.votes);
-          return [
-            h(TableTextButton, {
-              key: `vote-button-${e.referenceId}`,
-              onClick: () => props.history.push(`access_review/${dar.referenceId}`),
-              label: 'Vote',
-              disabled: isEmpty(votes)
-            }),
-            h(TableIconButton, {
-              icon: Block,
-              key: `cancel-button-${e.referenceId}`,
-              onClick: () => cancelElectionHandler(electionInfo, dar.referenceId, index)
-            })
-          ];
+          const voteButton = h(TableTextButton, {
+            key: `vote-button-${e.referenceId}`,
+            onClick: () => props.history.push(`access_review/${dar.referenceId}`),
+            label: 'Vote',
+            disabled: isEmpty(votes)
+          });
+          if (chairDacIds.includes(dacId)) {
+            return [
+              voteButton,
+              h(TableIconButton, {
+                icon: Block,
+                key: `cancel-button-${e.referenceId}`,
+                onClick: () => cancelElectionHandler(electionInfo, dar.referenceId, index)
+              })
+            ];
+          } else {
+            return [voteButton];
+          }
         }
         default :
           return h(TableTextButton, {
             key: `reopen-button-${e.referenceId}`,
             onClick: () => openConfirmation(dar, index),
-            label: 'Re-Open'
+            label: 'Re-Open',
+            disabled: !chairDacIds.includes(dacId)
```

Comment:
Since we're using this same logic in multiple places, it makes sense to pull this calculation out into a constant and just re-use the constant for disabled state and/or rendered state.